src/js/node/events.js

```diff
@@ -92,9 +92,11 @@ const emitWithoutRejectionCapture = function emit(type, ...args) {
   if (events === undefined) return false;
   var handlers = events[type];
   if (handlers === undefined) return false;
-
-  for (var handler of [...handlers]) {
```

Comment:
The use of `for-of` and all the spreading and stuff isn't great for JS perf hot spots.